examples/with-apollo-auth/lib/withApollo.js

```diff
@@ -69,7 +70,8 @@ export default App => {
 
       return {
         ...appProps,
-        apolloState
+        apolloState,
+        token
```

Comment:
This is potentially dangerous as passing down the cookie means you're taking input from the user and returning it to them, which, if Next.js didn't htmlescape the getInitialProps result, would lead to an XSS vulnerability.

Reply:
But, do cookies have a similar behavior maybe in a lower level? Cookies travel in the headers to the both ways. Does this represent an XSS vulnerability too?